src/test/java/org/mule/test/http/functional/listener/intercepting/cors/runner/CorsHttpEndpoint.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.http.functional.listener.intercepting.cors.runner;
+
+import org.mule.modules.cors.endpoint.KernelTestEndpoint;
+import org.mule.tck.junit4.rule.DynamicPort;
+
+public class CorsHttpEndpoint implements KernelTestEndpoint {
+
+  private final String path;
+  private final DynamicPort port;
+
+  public CorsHttpEndpoint(String path, DynamicPort port) {
```

Comment:
No, but I thought it made more sense than the port being a String... It does couple implementation to DynamicPort and that's not good. Already changed.